src/main/java/com/commercetools/helper/mapper/PaymentMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.commercetools.helper.mapper;
+
+import com.commercetools.model.CtpPaymentWithCart;
+import com.paypal.api.payments.Payment;
+
+import javax.annotation.Nullable;
+
+public interface PaymentMapper {
+
+    /**
+     * See Chapter <b><code>04. Payments</code></b> of <i>Integrating PayPal PLUS</i> guide.
+     *
+     * @param ctpPaymentWithCart commercetools cart and payment to map
+     * @return Paypal Plus {@link Payment} with respective values from {@link CtpPaymentWithCart}
+     * @see <a href="https://www.paypalobjects.com/webstatic/downloads/PayPal-PLUS-IntegrationGuide.pdf">
+     * Integrating PayPal PLUS</a>
+     */
+    @Nullable
+    Payment ctpPaymentToPaypalPlus(@Nullable CtpPaymentWithCart ctpPaymentWithCart);
```

Comment:
IMHO - not better in this certain case. maybe i'll change in the future to avoid @Nullable at all.